docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 
+After you complete this tutorial, you will be ready to apply more advanced security mechanisms in Quarkus, for example, how to use the OpenID Connect (OIDC) authentication mechanism.
 
 == Prerequisites
 
 include::{includes}/prerequisites.adoc[]
 
-== Architecture
+== What you will build
 
-In this example, we build a very simple microservice which offers three endpoints:
+The steps in this tutorial guide you through building an application that provides the following endpoints:
 
-* `/api/public`
-* `/api/users/me`
-* `/api/admin`
+[cols="20%,40% ",options="header"]
+|===
+|Endpoint | Description 
+|`/api/public`| The `/api/public` endpoint can be accessed anonymously.
+|`/api/users/me`|The `/api/admin` endpoint is protected with role-based access control (RBAC), and only users who have been granted the `admin` role can access it. 
+At this endpoint, the `@RolesAllowed` annotation is used to declaratively enforce the access constraint.
+| `/api/admin`| The `/api/users/me` endpoint is protected with RBAC and only users that have been granted the `user` role can access it. A JSON document with details about the user is returned as a response.
+|===
 
-The `/api/public` endpoint can be accessed anonymously.
-The `/api/admin` endpoint is protected with RBAC (Role-Based Access Control) where only users granted with the `admin` role can access. At this endpoint, we use the `@RolesAllowed` annotation to declaratively enforce the access constraint.
-The `/api/users/me` endpoint is also protected with RBAC (Role-Based Access Control) where only users granted with the `user` role can access. As a response, it returns a JSON document with details about the user.
+[TIP]
+====
+If you just want to examine the code, you can fast-forward to the completed example by using one of the following ways:
 
-== Solution
+* Download the {quickstarts-archive-url}[archive]
+* Clone the Git repository: `git clone {quickstarts-clone-url}`
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
+You can find the solution in the `security-jpa-quickstart` {quickstarts-tree-url}/security-jpa-quickstart[directory].
+====
 
-Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+:sectnums:
+:sectnumlevels: 3
 
-The solution is located in the `security-jpa-quickstart` {quickstarts-tree-url}/security-jpa-quickstart[directory].
+== Create a Maven project with the `security-jpa` extension
 
-== Creating the Maven Project
+For Quarkus security to be able to map your security source to JPA entities, ensure that the Maven project that is used in this tutorial includes the `security-jpa` extension.
+You can either create a new Maven project with the `security-jpa` extension or you can add the extension to an existing Maven project.
```

Comment:
Now I'm thinking about this (why I didn't think about this the first 6 times I read it.. 🙄) .. should we be simplifying here? Tutorial is a single path. Should we just state flat out: create a new project? (then we can skip the add extension to existing project.. variant.. )

Reply:
FIxed: Heading simplified.